pkg/queue/sharedmain/main.go

```diff
@@ -59,7 +60,7 @@ const (
 	// Duration the /wait-for-drain handler should wait before returning.
 	// This is to give networking a little bit more time to remove the pod
 	// from its configuration and propagate that to all loadbalancers and nodes.
-	drainSleepDuration = 30 * time.Second
+	drainSleepDuration = 15 * time.Second
```

Comment:
any reason why you lowered this?

Reply:
I will revert, was change I made internally